src/elements/card/src/index.tsx

```diff
@@ -22,14 +24,16 @@ const makeStyles = (variant: string) => (element?: string) =>
 
 const Card: React.FC<Props> = ({
   className = '',
+  date,
   description,
   horizontal = false,
   imageSize = 'cover',
   imgAlt,
   imgSizes = '768px',
   imgSrc,
   linkHref,
-  linkText = 'Read more',
+  linkText,
+  tag,
```

Comment:
as i said in my other comment, this is possibly tags as we could have articles with multiple tags maybe, lets check with rob